ci/tox/local-run.sh

```diff
@@ -10,5 +10,7 @@ check_var GIT_VERSION
 check_var PYTHON_VERSION
 
 set -x
-docker-compose build --build-arg user_id="$(id -u)" --build-arg group_id="$(id -g)" tox
-docker-compose run tox
+export TARGET=LOCAL
+export MOUNT_POINT=home/ci-user
+COMPOSE_DOCKER_CLI_BUILD=1 DOCKER_BUILDKIT=1 docker-compose build --build-arg user_id="$(id -u)" --build-arg group_id="$(id -g)" tox
```

Comment:
Awww okay, `DOCKER_BUILDKIT` is needed for multi-stage build in Dockerfile, and `COMPOSE_DOCKER_CLI_BUILD` just tells `docker-compose` to use `docker` CLI rather than some default builder, which wouldn't support multi-stage builds 🤯

Reply:
only:
```
export TARGET=LOCAL
export MOUNT_POINT=home/ci-user
```
is needed

Reply:
circleci has right settings by default to make it work

Reply:
and as it turns out locally docker also has right settings so there is no need for it